test/fixtures/config_v0_arc_cluster_point.js

```diff
@@ -481,6 +481,9 @@ export const parsedConfigV0 = {
       },
       brush: {
         enabled: false
+      },
+      geocoder: {
```

Comment:
this is for backward compatibility test, you should not add `geocoder` in here. The idea is when loading an older version of config which doesn't contain geocoder, loading should not faill, and geocoder should be diabled by default